components/typography/test/styles.test.js

```diff
@@ -25,12 +25,14 @@ describe.only('_isValidCssSelector', () => {
 		expect(_isValidCssSelector('p[loading]')).to.be.true;
 	});
 
-	it('should not support complex selectors', () => {
-		expect(_isValidCssSelector('dl dd')).to.be.false;
-		expect(_isValidCssSelector('dt.class')).to.be.false;
-		expect(_isValidCssSelector('dt#id')).to.be.false;
-		expect(_isValidCssSelector('dl .class')).to.be.false;
-		expect(_isValidCssSelector('dl #id')).to.be.false;
+	it('should support complex selectors', () => {
+		expect(_isValidCssSelector('dl dd')).to.be.true;
+		expect(_isValidCssSelector('dt.class')).to.be.true;
+		expect(_isValidCssSelector('dt#id')).to.be.true;
+		expect(_isValidCssSelector('dl .class')).to.be.true;
+		expect(_isValidCssSelector('dl #id')).to.be.true;
+		expect(_isValidCssSelector('dl > dt')).to.be.true;
+		expect(_isValidCssSelector('dl > dt > dd')).to.be.true;
 	});
```

Comment:
Do you think we should add a test that explicitly tests something like `:host ...` or `:host(...) ...`?

Reply:
Hmm yeah, good call.

This does remind me though, that  I didn`t add `:` to my admittedly not-very-smart regex. (But I think that's okay?).

Reply:
I can't imagine them wanting to target with pseudo-class or this - seems like a weird case that they'd want special typography applied only when a pseudo-class is in play. I'm not sure if it would ever come up, but in theory they might target a pseudo-element, for example `::before` or `::after`.

Reply:
Yeah, pseudo-classes + this dynamic css generation seems very niche and something we could tackle if it ever came up.

Reply:
But ultimately we only _really_ want this used when absolutely needed. Consumers hopefully won't even need these generator functions, so if we want to exclude the `:` cases for now, I think that's fine.